std_msgs/Empty req
---
std_msgs/String[] instances